content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -145,32 +147,34 @@ the version.
 
 ## Webhook conversion
 
-The above example has a None converter between versions which only set `apiVersion` field
-on conversion and do not change the rest of the object. API server supports webhook conversion that
+{{< note >}}
+Webhook conversion is intorduced in kubernetes 1.13 as an alpha feature. To use this feature, the
+`CustomResourceWebhookConversion` feature get should be set. Please refer to [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
+{{< /note >}}
+
+The above example has a None conversion between versions which only sets the `apiVersion` field
+on conversion and does not change the rest of the object. apiserver also supports webhook conversion that
 calls an external service in case of a conversion is required, e.g. when:
 
-* CR is requested in a different version than stored version.
-* CR list request contains objects of different versions than the request version.
+* custom resource is requested in a different version than stored version.
 * Watch is created in one version but the changed object is stored in another version.
-* CR PUT request is in a different version than storage version.
+* custom resource PUT request is in a different version than storage version.
 
-To cover all of these cases and to optimize conversion on API Server side, the conversion requests
-comes with a list of objects to be converted. These conversions should happened independent on these
-objects because grouping those CRs together in one request could have no meaning other than optimization.
+To cover all of these cases and to optimize conversion on apiserver side, the conversion requests carry as many object as possible in order to minimize the calls to the webhook. These conversions should happened independently. The grouping of custom resources does not represent any relation between them.
```

Comment:
and to optimize conversion by the apiserver, the conversion requests contain as many objects as possible ...